### ISSUE TITLE
Wrong information regarding cms 2.4.x compatibility in readme

### ISSUE THREAD

**Issue by stefanfoulis** [aldryn/aldryn-newsblog#214]
I'm not 100%, but I think this text in the readme is wrong:

```
*FROM VERSION 1.0.0, ALDRYN_NEWSBLOG WILL REQUIRE CMS 3.0 OR LATER*

If you intend to migrate from a 2.x project, please make sure you first
upgrade your project to the latest CMS 3.0.x, run all South migrations,
then, you can upgrade to futher CMS releases (3.1.x, etc.)
```

I think the table renaming has already happened in the south migrations, so it's already not compatible with django cms 2.4.x anymore (as of ~v0.9).

**Comment by mkoistinen** [comment]
Hmmm, fair point.

**Comment by mkoistinen** [comment]
Closed by #235.
